commerce_coordinator/apps/ecommerce/signals.py

```diff
@@ -0,0 +1,10 @@
+"""
+Ecommerce signals and receivers.
+"""
+import logging
+
+from commerce_coordinator.apps.core.signal_helpers import CoordinatorSignal
+
+logger = logging.getLogger(__name__)
```

Comment:
Logger is here for future use?

Reply:
Yep!

Reply:
Is that necessary? Wouldn't it be better to introduce them when we actually need them

Reply:
We can. I will remove them in a subsequent PR!